### PR TITLE
Carlyto

### PR DESCRIPTION
**Pull request by Carl2678**
hello-microverse

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (vickymarz) [inline .gitignore:1]: - [ ]  Kindly ensure that you add` test.md` file to the .gitignore just as the previous reviewer suggested. Note that once the test.md file is ignored, it should be gone from your repository.

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @Carl2678, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯

- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Good job so far 👍 As the previous reviewer has earlier said, you added a few sections of the [Template provided by Microverse](https://github.com/microverseinc/readme-template) and deleted the rest of the informative sections. Please add all the required information to each section of the [provided Template](https://github.com/microverseinc/readme-template) to make your repository professional 😄.

- [ ] KIndly ensure that you always write a well descriptive branch name that corresponds to the feature you are working on, instead of writing `carlyto`, you can use `hello-microverse`  or `project-setup` which is more professional 🤓 

- [ ] KIndly note that you are not supposed to create another Pull request containing changes from the previous review. WHen you are given a feedback, kindly implement the feedback in the same PR instead of creating a new one 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Carl2678 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Good commit messages. :heavy_check_mark:


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please your README file is not professional enough alot of details are missing. Check this [microverse Readme file template](https://github.com/microverseinc/readme-template) and kindly make the necessary adjustments. 😄 
- [ ] Then, please update you .gitignore file to ignore test.md as the formal reviewer suggested as it is, your .gitignore file is empty. 😄 

## Optional suggestions

- [Optional] I am sorry that the formal reviewer did not mention this; kindly give your PR a descriptive title. You can reference the image below for clarity. 😄 

https://user-images.githubusercontent.com/52934128/177452738-0610c008-a53a-454e-8aac-635cb4ed812c.gif


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @Carl2678, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
✅ Linters is passing

 ## Required Changes ♻️
_Check the comments under the review._
I couldn't use inline comments so I am including the suggestion here
[REQUIRED] You did not use the Microverse README template as suggested by previous reviewers. You can check an example in this [repo](https://github.com/fobadara/gcxchange/blob/development/README.md)

## Optional suggestions
[OPTIONAL] You should also have node_modules in your .gitignore file but the previous reviewers missed it. 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fherrerao — APPROVED** [review]
# ✅  🎆  STATUS APPROVED! ✅ 🎆 
#
Hi @Carl2678, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great Job Yes GIF by Hollywood Suite


## Highlights
✔️  No linter errors.
✔️  You used the correct GitHub-Flow.
✔️  Good README
✔️  Good Commit Messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] [OPTIONAL] Please don't forget to edit the branch name 🦖, the branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section') 🦮.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**